rocketpool-cli/node/stake-rpl.go

```diff
@@ -323,3 +346,26 @@ func nodeStakeRpl(c *cli.Context) error {
 	return nil
 
 }
+
+func borrowedRplStake(borrowedPerMinipool *big.Int, rplPrice *big.Int) *big.Int {
+	percentBorrowedRplStake := big.NewInt(0)
+	percentBorrowedRplStake.Mul(eth.EthToWei(24), borrowedPerMinipool)
+	percentBorrowedRplStake.Div(percentBorrowedRplStake, rplPrice)
+	percentBorrowedRplStake.Add(percentBorrowedRplStake, big.NewInt(1))
+	amountWei := percentBorrowedRplStake
+
+	return amountWei
+
+}
+
+func parsePercentageInput(customInput string, rplPrice *big.Int) (*big.Int, error) {
+	trimmedInput := strings.TrimSuffix(customInput, "%")
+	stakePercent, err := strconv.ParseFloat(trimmedInput, 64)
```

Comment:
this is fine but just for edification purposes, [fmt.Sscanf](https://pkg.go.dev/fmt#Sscanf) may be more semantic, ie https://go.dev/play/p/BVofq8L4EpC